tests/CalendarApp.UnitTests/MeetingRemoverTests.cs

```diff
@@ -0,0 +1,15 @@
+using System;
```

Comment:
where tests? :)

Reply:
Ignore it, I forgot to delete that file, my apologies 🙈🙏